src/containers/Messages/Messages.js

```diff
@@ -68,15 +79,28 @@ const Messages = () => {
     firebase.auth().onAuthStateChanged(async function(user) {
       if(!user) return window.location.href = '/login';
 
-      const admin = await isAdmin(firebase);
+      const admin = await checkIsAdmin(firebase);
       const roomOwner = user.uid === room;
       if(!admin && !roomOwner) return window.location.href = '/login';
 
+      setIsAdmin(admin);
```

Comment:
To add admin id in receiver key in the message which will be required to get the device token of admin for sending the notification.

Since admin is not identified anywhere and having a hard time to get admin id since there could be more that one admin so I am adding admin id in conversation when any of the admin joined the converstion.